configs/src/deployments/json/holesky.json

```diff
@@ -0,0 +1,179 @@
+{
+  "v1.3.0": {
+    "AddresslistVotingSetup": {
+      "address": "0x07f49c49Ce2A99CF7C28F66673d406386BDD8Ff4",
+      "blockNumber": 1842649,
+      "deploymentTx": "0xdb97cb5667417edd33f1a0b17974a8b5e5cec88e6d806448f000eee75c155523"
+    },
+    "AddresslistVotingSetupImplementation": {
+      "address": "0x917C2Ab96c40aDEfD08d240409485D8b606423E3",
+      "blockNumber": 1842649,
+      "deploymentTx": "0xdb97cb5667417edd33f1a0b17974a8b5e5cec88e6d806448f000eee75c155523"
+    },
+    "AdminSetup": {
+      "address": "0x5B3B36BdC9470963A2734D6a0d2F6a64C21C159f",
+      "blockNumber": 1842653,
+      "deploymentTx": "0xacc3afcb6abd50186f5c4b56b638ef34f1edc44dce9bf2e4e3115688b9bb587f"
+    },
+    "AdminSetupImplementation": {
+      "address": "0x30fA1EF45Ac5cd56454fbc1B5Aa832a70d9Da166",
+      "blockNumber": 1842653,
+      "deploymentTx": "0xacc3afcb6abd50186f5c4b56b638ef34f1edc44dce9bf2e4e3115688b9bb587f"
+    },
+    "DAOFactory": {
+      "address": "0xE640Da5AD169630555A86D9b6b9C145B4961b1EB",
+      "blockNumber": 1842644,
+      "deploymentTx": "0x840b93e6cc41ae48bb27a609ade72ba6c0df3f6d7928473ff2a0fcec450c56a8"
+    },
+    "DAORegistryProxy": {
+      "address": "0x005098056a837c2c4F99C7eCeE976F8D90bdFFF8",
+      "blockNumber": 1842639,
+      "deploymentTx": "0x0eb4572e09f3e87b002d6e72069440969fc1fce31b25c804f4ce75751c078510"
+    },
+    "DAORegistryImplementation": {
+      "address": "0x6115b7F05cCeF2D883AB64c88ba7e4BD1b877215",
+      "blockNumber": 1842638,
+      "deploymentTx": "0xd4448e0298947bcb8563ebb3d3d8fb1b7f39e5632fe4a6b14a635a7538601b76"
+    },
+    "DAOENSSubdomainRegistrarProxy": {
+      "address": "0x5ec029bE8A0d322a54bb9C672cc0feB7dDD5C88F",
+      "blockNumber": 1842633,
+      "deploymentTx": "0x59c1f3e45c851ac671273d50e76c63aafe8134f29d0400421a4bcdcba9d78242"
+    },
+    "DAOENSSubdomainRegistrarImplementation": {
+      "address": "0x6d4FB6Ff01A172774f42789fcfcdd84E68c28494",
+      "blockNumber": 1842632,
+      "deploymentTx": "0x8c489dd5f838070daad7ba3691240b46b341270b17f5287d4b87028200b8a17f"
+    },
+    "ENSRegistry": {
+      "address": "0x6d4FB6Ff01A172774f42789fcfcdd84E68c28494",
+      "blockNumber": 1842632,
+      "deploymentTx": "0x8c489dd5f838070daad7ba3691240b46b341270b17f5287d4b87028200b8a17f"
+    },
```

Comment:
this is the Registrar address

Reply:
Holesky  supports ENS so this address is not needed since we are not deploying our own

Reply:
addressed at https://github.com/aragon/osx-commons/pull/92/commits/2850c12361340a1e0f5e186801ff58b1a58228c4